app/models/paye/PayeAtsData.scala

```diff
@@ -154,8 +154,8 @@ case class PayeAtsData(
         calculatedTotals.flatMap(_.totalIncomeTax)
       }
     )
-
-    GovSpendingDataTransformer(totalIncome, taxYear).govSpendReferenceDTO
+    val correctYearForGovSpendCategories = taxYear + 1
+    GovSpendingDataTransformer(totalIncome, correctYearForGovSpendCategories).govSpendReferenceDTO
```

Comment:
Missing a test, how did this not break other tests??

Reply:
yes agreed ..no tests were written to cover this scenario
added one now